mac

```diff
@@ -301,7 +304,7 @@ append_web_dependencies() {
     brew "imagemagick"
 
     # Databases
-    brew "postgres"
+    brew "postgresql@14"
```

Comment:
It is worth to do option from "Potential improvements" in this PR:

> Make a global variable for postgresql version number

Reply:
I thought about it but wasn't 100% sure the version number would directly map to the folder paths in the future, and unsure if we will remove/refactor the `setup_postgres` where it's used 3 out of the 4 times.

Reply:
Ok